stocks/stocks.py

```diff
@@ -1,7 +1,155 @@
-from flask import Blueprint
+import requests
+import os
+import json
+import sqlite3 as sql
+import config
 
-stocks_bp = Blueprint('stocks_bp', __name__)
+from flask import Blueprint, request
+from dotenv import load_dotenv
 
-@stocks_bp.route('/')
+
+load_dotenv()
+
+FINNHUB_TOKEN = os.getenv("FINNHUB_TOKEN")
+
+STOCK_QUOTE_URL = "https://finnhub.io/api/v1/quote?token={token}&symbol={symbol}"
+STOCK_PROFILE_URL = "https://finnhub.io/api/v1/stock/profile2?token={token}&symbol={symbol}"
+
+stocks_bp = Blueprint("stocks_bp", __name__)
+
+@stocks_bp.route("/example")
 def index():
-    return "This is an example stock app"
\ No newline at end of file
+    return "This is an example stock app"
+
+def get_stock_quote(symbol):
+    """
+    Get real-time quote data for a stock given symbol
+    """
+    r = requests.get(STOCK_QUOTE_URL.format(token=FINNHUB_TOKEN, symbol=symbol))
+    response = r.json()
+    return response
+    """
+        {
+            "c": 261.74,
+            "h": 263.31,
+            "l": 260.68,
+            "o": 261.07,
+            "pc": 259.45,
+            "t": 1582641000 
+        }
+    """
+
+def validate_stock_symbol(symbol):
+    """
+    Make sure symbol is trackable
+    """
+    r = requests.get(STOCK_PROFILE_URL.format(token=FINNHUB_TOKEN, symbol=symbol))
+    response = r.json()
+    if response: 
```

Comment:
What does response look like if the symbol doesn't exist?

If you really want to return two types, you can just do:
```
return response.get("name", False)
```

Reply:
switched to:
`if response: 
        return response.get("name")
    return `

if symbol doesn't exist, response = {}

Reply:
Since symbol doesn't exist is {}, the suggestion will work:

`return response.get("name")`

Get returns None when "name" can't be found